src/content/init.js

```diff
@@ -10,48 +10,51 @@ import './index.css';
  */
 export async function init(getSettings) {
     let json;
-    let raw;
 
-    const { firstElementChild } = document.body;
-    let { textContent } = document.body;
+    const firstElementChild = document.body.firstElementChild;
+    if (!firstElementChild || firstElementChild.tagName !== 'PRE') {
+        return;
+    }
 
-    textContent = textContent.trim();
+    let textContent = firstElementChild.textContent.trim();
+    if (!textContent.startsWith('{') && !textContent.startsWith('[')) {
+        return;
+    }
 
-    if (
-        (firstElementChild && firstElementChild.tagName === 'PRE') &&
-        (textContent.startsWith('{') || textContent.startsWith('['))
-    ) {
-        try {
-            json = JSON.parse(textContent);
-        } catch (_) {}
+    try {
+        json = JSON.parse(textContent);
+    } catch (e) {
+        console.error(e.message);
```

Comment:
To prevent linter from complaining to this console statement, please add a comment `// eslint-disable-line no-console`